ttnn/CMakeLists.txt

```diff
@@ -93,6 +93,8 @@ set(TTNN_PUBLIC_LINK_LIBRARIES compiler_flags metal_header_directories metal_com
 set(TTNN_PUBLIC_LINK_DIRS "")
 
 set(TTNN_PRECOMPILED_HEADERS
+    ${PROJECT_SOURCE_DIR}/tt_metal/tt_stl/reflection.hpp
+    ${PROJECT_SOURCE_DIR}/ttnn/cpp/ttnn/operation.hpp
```

Comment:
@ayerofieiev-tt @eyonland 
Do we want this ?

Reply:
I am okay with it. Changes to these headers will anyway cause major rebuild